### PR TITLE
Fixed links and terms

### LINKED ISSUE #184: Fix links and terms
A few mistakes found their way into production. Here's a plan for a relatively easy, quick fix: 
- footer on beta site
  - [ ] point "privacy" link to: https://www.mozilla.org/en-US/privacy/websites/
  - [ ] point "legal" link to: https://webmaker.org/en-US/terms
  - we will redirect "legal" to the new terms page on beta when they go into effect
- legal terms page on beta site
  - [ ] add to the top "Below are the new terms of service for Webmaker and Mozilla Learning 
    Networks, which will go into effect July 27, 2015"
  - note: nothing should link here until terms are finalized and announced

### PR DESCRIPTION
**Pull request by mmmavis**
This fixes #184 
- [x] remove Privacy page from repo
- [x] point "privacy" link on footer to: https://www.mozilla.org/en-US/privacy/websites/
- [x] point "legal" link on footer to: https://webmaker.org/en-US/terms 
- [x] keep `/#/legal` alive and update terms copy (copy is still WIP that's why we want to unlink this page from footer for now)

### CODE REVIEW

File: src/pages/legal/tos.jsx [hunk 1]
```diff
@@ -67,11 +66,11 @@ module.exports = React.createClass({
 
             <p>We may suspend or terminate your accounts or cease providing you with all or part of the Sites at any time for any reason, including, but not limited to, if we reasonably believe: (i) you have violated these Terms, (ii) you create risk or possible legal exposure for us; or (iii) our provision of the Sites to you is no longer commercially viable. We will make reasonable efforts to notify you by the email address associated with your account or the next time you attempt to access your account.</p>
 
-            <p>In all such cases, the Terms shall terminate, except that the following sections shall continue to apply: Release and Indemnification, Disclaimer; Limitation of Liability, Governing Law, Miscellaneous.</p>
+            <p>n all such cases, the Terms shall terminate, except that the following sections shall continue to apply: Release and Indemnification, Disclaimer; Limitation of Liability, Governing Law, Miscellaneous.</p>
```
Comment (vazquez) [inline src/pages/legal/tos.jsx]: Missing the letter 'I' here. `In all such cases`

**Comment by vazquez** [comment]
Links are working :) Everything good except for the itty bitty typo.

Other than that, R+ :cat:

File: src/pages/legal/tos.jsx (hunk 1 above)
Comment (mmmavis) [inline src/pages/legal/tos.jsx]: wowow nice catch :dancers:

**Comment by xmatthewx** [comment]
thanks for the speedy fixes! one more typo: ~~June~~ 27 --> July 27

**Comment by mmmavis** [comment]
oops, i'll open another ticket for it

**Pull request merged**